SparkPost/support-docs
# WE-410: relative image links breaking on initial load

### What Changed

Fixes the image error, adds some tests and fixes a unique key error I was getting sometimes.

### How To Test or Verify

- Navigate to the following pages:
   - /docs/deliverability/deliverability-analytics/
   - /docs/billing/upgrading-your-account/
   - /docs/reporting/health-score/
- Verify that all the images load on initial page load

### PR Checklist
